### DISCUSSION TITLE
PCGamingWiki Plugin Request

### DISCUSSION THREAD

**Discussion by brianx87** [SteamDeckHomebrew/decky-loader#216] [Ideas]
Could someone possibly make a plugin for PCGamingWiki similar to that of the ProtonDB plugin?

PCGamingWiki has useful information about game data, mods and tweaks that would be really useful to access from gaming mode.

**Comment by TrainDoctor** [comment]
All developers for decky-loader and all plugin developers are volunteers and we don't accept requests for plugins. I love PCGamingWiki and would love to see an addon like what you suggested but I'll close this issue as it's a matter of policy rather than content. Cheers.
